### PR TITLE
Add webhook name and room to admin info

### LINKED ISSUE #619: More info in the „You have bridged a webhook“ message
Use cases are for example looking up URLs for webhooks or to know when and where I configured them.

I would suggest changing

```
You have bridged a webhook. Please configure your webhook source to use https://example.com/
```

to

```
You have bridged the webhook „hook name“ in #room:example.com.
Please configure your webhook source to use https://example.com/
```

### PR DESCRIPTION
**Pull request by weeman1337**
Now in admin webhook info:
- Name of the hook
- Link to room (used a matrix.to link, because that seems to be working)
- URL on own line for better copying

image

closes #619

### CODE REVIEW

File: src/Connections/SetupConnection.ts [hunk 1]
```diff
@@ -216,7 +216,12 @@ export class SetupConnection extends CommandConnection {
         this.pushConnections(c.connection);
         const url = new URL(c.connection.hookId, this.config.generic.parsedUrlPrefix);
         const adminRoom = await this.getOrCreateAdminRoom(userId);
-        await adminRoom.sendNotice(`You have bridged a webhook. Please configure your webhook source to use ${url}.`);
+        await adminRoom.sendNotice(
+            `You have bridged the webhook "${name}" in https://matrix.to/#/${this.roomId} .\n` +
```
Comment (jaller94) [inline src/Connections/SetupConnection.ts:220]: While an official Synapse will generate safe room IDs, the spec doesn't seem to limit what characters might be included.

matrix.to can deal with URL-encoded room IDs and room aliases.

```suggestion
            `You have bridged the webhook "${name}" in https://matrix.to/#/${encodeURIComponent(this.roomId)} .\n` +
```

Alternatively, if you think it looks too weird that ":" gets encoded:
```suggestion
            `You have bridged the webhook "${name}" in https://matrix.to/#/${encodeURIComponent(this.roomId).replace(/%3A/g, ":")} .\n` +
```

File: src/Connections/SetupConnection.ts [hunk 2]
```diff
@@ -216,7 +216,12 @@ export class SetupConnection extends CommandConnection {
...
+                // Line break before and no full stop after URL is intentional.
+                // This makes copying and pasting the URL much easier.
```
Comment (jaller94) [inline src/Connections/SetupConnection.ts:222]: You may have these 8 spaces back to put them somewhere else. :yum: 
```suggestion
            // Line break before and no full stop after URL is intentional.
            // This makes copying and pasting the URL much easier.
```

**Review by jaller94 — APPROVED** [review]
Looks good to me. Accepting the suggestions would be highly appreciated.

File: src/Connections/SetupConnection.ts (hunk 1 above)
Comment (Half-Shot) [reply src/Connections/SetupConnection.ts:220]: Former looks fine to me :)

File: src/Connections/SetupConnection.ts (hunk 2 above)
Comment (weeman1337) [reply src/Connections/SetupConnection.ts:222]: Since we started using Prettier in other repos, I have become really lazy :upside_down_face:

**Pull request merged**